Crash more gracefully if build workflow is not found

In a repo where amalgamate-pages was run but no build workflow exists:

```
+ 14:19:14 INFO root: Finding latest release with an asset whose name ends with '-web.zip'
+ 14:19:14 INFO root: No suitable release/asset found
Traceback (most recent call last):
  File "/home/runner/work/_actions/endlessm/amalgamate-pages/v1/godoctopus.py", line 416, in <module>
    main()
  File "/home/runner/work/_actions/endlessm/amalgamate-pages/v1/godoctopus.py", line 412, in main
    amalgamate_pages.run()
  File "/home/runner/work/_actions/endlessm/amalgamate-pages/v1/godoctopus.py", line 271, in run
    workflow = self.find_workflow()
               ^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/_actions/endlessm/amalgamate-pages/v1/godoctopus.py", line 114, in find_workflow
    raise ValueError(f"Workflow '{self.workflow_name}' not found")
ValueError: Workflow 'Build and Export Game' not found
Error: Process completed with exit code 1.
```

What is shown at the top level is just:

<img width="477" height="164" alt="Image" src="https://github.com/user-attachments/assets/6e4c373f-8e33-4e1b-85eb-7f3dda9ccfc3" />

Can we improve that to show the error more clearly?

--- a/godoctopus.py
+++ b/godoctopus.py
@@ -16,6 +16,10 @@ import requests
 import requests_cache
 
 
+class ConfigurationError(Exception):
+    pass
+
+
 @dataclasses.dataclass
 class Build:
     workflow_run: dict
@@ -120,7 +124,10 @@ class AmalgamatePages:
             if workflow["name"] == self.workflow_name:
                 return workflow
 
-        raise ValueError(f"Workflow '{self.workflow_name}' not found")
+        raise ConfigurationError(
+            f"Workflow '{self.workflow_name}' not found. "
+            "Has this project been built at least once?"
+        )
 
     def list_branches(self, repo: str) -> list[dict]:
         try:
@@ -417,8 +424,12 @@ def main() -> None:
 
     session = make_session(api_token)
 
-    amalgamate_pages = AmalgamatePages(session, repo, workflow_name, artifact_name)
-    amalgamate_pages.run()
+    try:
+        amalgamate_pages = AmalgamatePages(session, repo, workflow_name, artifact_name)
+        amalgamate_pages.run()
+    except ConfigurationError as e:
+        print(f"::error::{e.args[0]}")
+        raise
 
     session.cache.delete(older_than=dt.timedelta(days=7))
 

